src/redux/auth/auth-operations.ts

```diff
@@ -32,11 +33,9 @@ const logIn = createAsyncThunk(
 
 const register = createAsyncThunk(
   "auth/register",
-  async (credentials, { dispatch, rejectWithValue }) => {
+  async (credentials:IUser, { dispatch, rejectWithValue }) => {
```

Comment:
Add whitespace after colon, please) Btw, do you know about prettier?

Reply:
done))